src/seedu/addressbook/AddressBook.java

```diff
@@ -1161,8 +1161,8 @@ private static String getUsageInfoForExitCommand() {
      * @param sign  Parameter sign to be removed
      * @return  string without the sign
      */
-    private static String removePrefixSign(String s, String sign) {
-        return s.replace(sign, "");
+    private static String removePrefix(String fullString, String prefix) {
+        return fullString.replaceFirst("^" + prefix, "");
```

Comment:
It will only replace the first instance because of the ^{prefix} regex which will only match a string that begins with the prefix.